modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc

```diff
@@ -0,0 +1,95 @@
+:_content-type: PROCEDURE
+:description: Starting a new workspace with a clone of a Git repository
+:keywords: start-new-workspace, start-a-new-workspace, how-to-start-new-workspace, how-to-start-a-new-workspace, starting-a-new-workspace, clone-git-repository, clone-a-git-repository, how-to-start-workspace, how-to-start-a-workspace
+:navtitle: Starting a new workspace with a clone of a Git repository
+// :page-aliases:
+
+[id="starting-a-new-workspace-with-a-clone-of-a-git-repository_{context}"]
+= Starting a new workspace with a clone of a Git repository
+
+Working with {prod-short} in your browser involves multiple URLs:
+
+* The URL of your organization's {prod-short} instance, used as part of all the following URLs
+* The URL of the *Workspaces* page of your {prod-short} dashboard with the workspace control panel
+* [.underline]#The URLs for starting a new workspace#
+* The URLs of your workspaces in use
+
+With {prod-short}, you can visit a URL in your browser to start a new workspace that contains a clone of a Git repository. This way, you can clone a Git repository that is hosted on GitHub, a GitLab instance, or a Bitbucket server.
+
+NOTE: The *Create Workspace* page of your {prod-short} dashboard has the *Git Repo URL ** field as the limited-functionality GUI alternative for entering the URL of a Git repository to start a new workspace.
+
+.Prerequisites
+
+* Your organization has a running instance of {prod-short}.
```

Comment:
```suggestion
* Your organization has a running instance of {prod-short}, and you know its URL.
```

Reply:
I solved it like this:

![image](https://user-images.githubusercontent.com/78564304/162481372-a8269b22-a339-443a-901e-9bdab67de56c.png)

I need to introduce the user-replaced value `che-fqdn` here.